starcounter-include.html

```diff
@@ -246,7 +240,7 @@
          * @see .stampComposition
          * @param  {String} compositionString stringified HTML for new Shadow Root
          */
-        StarcounterIncludePrototype._compositionChanged = function(compositionString) {
```

Comment:
Renaming of `_compositionChanged` method will cause current CompositionEditor to be invalid. Can you keep the old name or at least have backwards compat with the old name?

Reply:
I'd prefer to reduce the number of backward compatible hacks in `sc-include`, and rather followup with simple change to CompositionEditor. I think old name was the reason was not matching the behavior and was a reason of confusion.

Reply:
Would

Reply:
Do you think, we should add tests for this aliasing, as well? I'd like to get rid of it really soon.

Reply:
No, I don't think we need tests for aliasing.